plugin/src/main/java/io/github/rysefoxx/content/InventoryProvider.java

```diff
@@ -41,6 +41,12 @@ public interface InventoryProvider {
     default void update(Player player, InventoryContents contents) {
     }
 
+    /**
+     * A method to close the inventory.
+     */
+    default void close(Player player) {
```

Comment:
```suggestion
    default void close(Player player, RyseInventory inventory) {
```

Since the interface method is called before the inventory is closed, I would still pass the RyseInventory as the 2nd parameter. Maybe the user wants to read values from the inventory before closing.